Update Windows install docs (#2977)
SFDO-Tooling/CumulusCI: docs/get_started.rst

****************

#. Go to the `Python downloads page <https://www.python.org/downloads/windows/>`_.
#. Download the latest Python 3.9 release. Most users select the "Windows Installer (64-bit)" link, but it depends on your particular computer setup.
#. Install using the installation wizard.

   * Select **Add Python <version> to PATH**.
   * Click "Install Now".

   .. image:: images/windows_python.png

> #. On the screen entitled "Setup was successful," you may see a button with the text "Disable path length limit". If you do, click it.

Reviewer: How about: 

```
#. On the screen entitled "Setup was successful," check the "Disable path length limit" box (if it's present).
```

> #. On the screen entitled "Setup was successful," you may see a button with the text "Disable path length limit". If you do, click it.

Reviewer: ```suggestion
#. On the screen entitled "Setup was successful," click the "Disable path length limit" button (if it's present).
```
